ui-tests/src/test/java/org/openmrs/reference/page/ManageUserAccountPage.java

```diff
@@ -0,0 +1,148 @@
+package org.openmrs.reference.page;
+
+import java.util.List;
+
+import org.openmrs.uitestframework.page.Page;
+import org.openqa.selenium.By;
+
+public class ManageUserAccountPage extends Page {
+
+    public static String URL = "/adminui/systemadmin/accounts/manageAccounts.page";
+    private static final By SEARCH_USER = By.cssSelector("#list-accounts_filter input[type=text]");
+    private static final By ADD_NEW_ACCOUNT = By.cssSelector("#content a[href='/openmrs/adminui/systemadmin/accounts/account.page']");
+    private static final By FAMILY_NAME = By.cssSelector("#adminui-familyName-field");
+    private static final By GIVEN_NAME = By.cssSelector("#adminui-givenName-field");
+    private static final By GENDER_FEMALE = By.cssSelector("#adminui-gender-1-field");
+    private static final By ADD_USER_ACCOUNT = By.id("adminui-addUserAccount");
+    private static final By USER_NAME = By.cssSelector("#adminui-username-field");
+    private static final By PRIVILEGE_LEVEL = By.cssSelector("#adminui-privilegeLevel-field");
+    private static final By PASSWORD = By.cssSelector("#adminui-password-field");
+    private static final By CONFIRM_PASSWORD = By.cssSelector("#adminui-confirmPassword-field");
+    private static final By ADD_PROVIDER_ACCOUNT = By.id("adminui-addProviderAccount");
+    private static final By IDENTIFIER = By.cssSelector("#adminui-identifier-field");
+    private static final By PROVIDER_ROLE = By.cssSelector("#adminui-providerRole-field");  
+    private static final By EDIT_USER_ACCOUNT = By.cssSelector("i.icon-pencil.edit-action");
+    private static final By ADMINISTER_SYSTEM = By.id("adminui-capabilities-Application: Administers System");
+    private static final By CONFIGURE_FORMS = By.id("adminui-capabilities-Application: Configures Forms");
```

Comment:
thanks @kdaud  just on a random check,what works better using the id as adminui-capabilities-Application or having : adminui-capabilities-Application: Configures Forms  hoping Configures Forms  never affects the behavior of the id

Reply:
@HerbertYiga I will move the ticket to READY FOR REVIEW when the commit is well constructed otherwise its not ready for review!